packages/search-service/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "search",
+  "description": "Search micro-service for One Platform",
+  "opservice": {
+    "className": "Search",
+    "serviceName": "search"
+  },
+  "version": "",
+  "keywords": [
+    "web-components",
+    "html"
```

Comment:
Change the keywords to something more appropriate. (eg. `graphql`, `microservice`, `search`, etc)